drivers/sensor/lsm6dsl/lsm6dsl.h

```diff
@@ -716,6 +716,16 @@ struct lsm6dsl_data {
 	int gyro_sample_z;
 #if defined(CONFIG_LSM6DSL_ENABLE_TEMP)
 	int temp_sample;
+#endif
+#if defined(CONFIG_LSM6DSL_EXT0_LIS2MDL)
+	int magn_sample_x;
+	int magn_sample_y;
+	int magn_sample_z;
+	float magn_sensitivity;
```

Comment:
Does this really need to be a float? Would an int be sufficient?

Reply:
OK. Hope to remember it...